UPGRADE-2.0.md

```diff
@@ -3,3 +3,47 @@
 ## UsesGuzzleHttpClient Trait & HttpClientInterface Interface
 
 * `HttpClientInterface` interface was added and `RestConfigurableProducer` implements it via the `UsesGuzzleHttpClient` trait. The interface adds an extra function `addHandler()` that allows users to add extra Handlers to the `ClientInterface`. By default a new handler is added (`Smartbox\Integration\FrameworkBundle\Components\WebService\Rest\Middleware`) while building the container fixing an bug in Guzzle that incorrectly splits multibyte charaters in two when truncating the response body. If you're already tweaking the Guzzle client used by the RestConfigurableProducer, you'll have to check if this new handler overrides it.
+
+
+## AMQP Support
+
+The AMQP driver was revamped **to use the [php-amqplib](https://github.com/php-amqplib/php-amqplib) library** instead of the PHP's built-in AMQP functions. The reason behind this was to take advantage of the non-blocking `consume()` function of this library, which the native driver doesn't support. The AMQP now does not ignore to posix signals what brings the possibility to stop the consumer gracefully with a `kill -s INT {pid}`. To facilitate this feature, now the consumer name in AMQP brings the PID number as the last part of the name.
```

Comment:
```suggestion
The AMQP driver was revamped **to use the [php-amqplib](https://github.com/php-amqplib/php-amqplib) library** instead of the PHP's built-in AMQP functions. The reason behind this was to take advantage of the non-blocking `consume()` function of this library, which the native driver doesn't support. Now the AMQP driver does not ignore posix signals bringing the possibility to stop the consumer gracefully with `kill -s INT {pid}`. To facilitate this feature, now the consumer name in AMQP shows the PID number as the last part of the name.
```